packages/server/src/api/routes/public/tests/users.spec.ts

```diff
@@ -1,26 +1,66 @@
-import * as setup from "../../tests/utilities"
+import TestConfiguration from "../../../../../src/tests/utilities/TestConfiguration"
 import { generateMakeRequest, MakeRequestResponse } from "./utils"
 import { User } from "@budibase/types"
 import { mocks } from "@budibase/backend-core/tests"
 
 import * as workerRequests from "../../../../utilities/workerRequests"
 
+function user() {
+  return {
+    _id: "user",
+    _rev: "rev",
+    createdAt: Date.now(),
+    email: "test@example.com",
+    roles: {},
+    tenantId: "default",
+    status: "active",
+  }
+}
+
+jest.mock("../../../../utilities/workerRequests", () => ({
+  getGlobalUsers: jest.fn(() => {
+    return {
+      _id: "us_uuid1",
+    }
+  }),
+  getGlobalSelf: jest.fn(() => {
+    return {
+      _id: "us_uuid1",
+    }
+  }),
+  allGlobalUsers: jest.fn(() => {
+    return [user()]
+  }),
+  readGlobalUser: jest.fn(() => {
+    return user()
+  }),
+  saveGlobalUser: jest.fn(() => {
+    return { _id: "user", _rev: "rev" }
+  }),
+  deleteGlobalUser: jest.fn(() => {
+    return { message: "deleted user" }
+  }),
+  removeAppFromUserRoles: jest.fn(),
+}))
```

Comment:
These used to live in a file almost everything imported, but they're only used here. Moved here to reduce confusion.